recipe/meta.yaml

```diff
@@ -24,18 +26,21 @@ requirements:
     - sed   # [not win]
     - gawk  # [not win]
     - cmake <=3.21
-    - patch  # [not win]
     - pkg-config  # [not win]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - patch  # [not win]
+    - m2-patch  # [win]
   host:
-    - zlib
-    - bzip2
-    - zstd
-    - lz4-c
-    - curl  # [not win]
-    - libcurl  # [not win]
-    - openssl  # [not win]
+    - zlib {{ zlib }}
+    - bzip2 {{ bzip2 }}
+    - zstd {{ zstd }}
+    - lz4-c 1.9.4
+    - curl 7.88.1 # [not win]
+    - libcurl 7.88.1  # [not win]
```

Comment:
Do we really need both curl and libcurl? Because curl has libcurl as a runtime dependency

Reply:
When I looked at it I had the feeling it was required. I'll double check.

Reply:
See https://github.com/AnacondaRecipes/tiledb-feedstock/blob/master/recipe/build.sh#L18.